Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,46 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
+.parent {
+  width: 300px;
+  height: 300px;
```

Comment:
There is a commented out line of code (padding: 80px 50px;) that should be removed.